src/util/ui_orders.ts

```diff
@@ -36,19 +48,21 @@ const ordersToUIOrdersWithoutOrderInfo = (orders: SignedOrder[], baseToken: Toke
             filled,
             price,
             status,
+            makerFillableAmountInTakerAsset: order.takerAssetAmount,
+            remainingTakerAssetFillAmount: order.takerAssetAmount,
```

Comment:
How come these are both assigned to `order.takerAssetAmount`?

Reply:
This is the case where there is no wallet so no on-chain Provider exists in this case. We can only assume the total order is fillable and the maker can fill it entirely. Without a wallet the user isn't able to interact and this is reloaded in the alternative flow once a wallet is connected.